### DISCUSSION TITLE
Missing "./calendar" specifier in "@rnwonder/solid-date-picker" package

### DISCUSSION THREAD

**Discussion by ngquyduc** [rnwonder/solid-date-picker#27] [Q&A]
When trying to use the `Calendar` component, I ran into this problem. 
![image](https://github.com/rnwonder/solid-date-picker/assets/72744234/f2723dfd-60d4-4467-bd04-26ce3c5c1bb9)

Here is the code I used (same as the example given [here](https://soliddatepicker.live/docs/installation/)) 
```
import Calendar from "@rnwonder/solid-date-picker/calendar";

const App = () => {

  return  (
    <Calendar
      onChange={(value) => {
        console.log('onChange', value)
      }}
      onValueChange={(value) => {
        console.log('onValueChange', value)
      }}
    />
  )
}
```

Have any one run into this before. If yes, what's your solution for this?

**Comment by rnwonder** [comment]
What version are you using? It should be available in the latest version

**Reply by ngquyduc** [reply]
Yes, I'm using the latest package. However, I just fixed it just now by delete the node_modules folder and reinstall everything. I think the problem comes from the fact that I use pnpm and first and then I use npm later to update the package.

**Reply by ngquyduc** [reply]
By the way, is this possible to restyle the Calendar as well as the Input. The style of the Calendar and my website currently doesn't match.
![image](https://github.com/rnwonder/solid-date-picker/assets/72744234/4b868391-e32d-47a8-8a8d-7d7be9387a10)

**Reply by rnwonder** [reply]
You should be able to, check the docs. There are styling props, classes ane data attributes you can use

**Reply by ngquyduc** [reply]
Thank you
